src/apps/applets/router.py

```diff
@@ -184,3 +185,12 @@
         **NO_CONTENT_ERROR_RESPONSES,
     },
 )(applet_link_delete)
+
+router.post(
+    "/{id_}/setRetention",
+    status_code=status.HTTP_200_OK,
+    responses={
+        **DEFAULT_OPENAPI_RESPONSE,
+        **AUTHENTICATION_ERROR_RESPONSES,
+    },
+)(applet_set_data_retention)
```

Comment:
I think it is enough to have `HTTP POST /applets/{id_}/retentions` endpoint. `set` prefix breaks out the consistency